Project/jobportal/index.css

```diff
@@ -0,0 +1,99 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    height: 2000px;
+}
+
+header.nav-header {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    background-color: #000;
+    color: #fff;
+    padding: 20px;
+    text-align: center;
+    display: flex;
+    justify-content: space-between;
+}
+
+nav ul {
+    list-style-type: none;
+    padding: 0;
+}
+
+nav ul li {
+    display: inline;
+    margin-right: 20px;
+}
+
+nav ul li a {
+    color: #fff;
+    text-decoration: none;
+}
+
+.banner {
+    margin: 100px 0px;
+    background-image: url('images/job-banner.jpeg');
```

Comment:
Please how other ways we can provide bg-images

Reply:
- Directly in html like:
`<p style="background-image: url('images/job-banner.jpeg');">`
- Directly from img:src and stacking elements over it 
`<img loading="lazy" class="search-icon" src="images/job-banner.jpeg" alt="search">`